kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -66,8 +66,21 @@
   private static final ConfigDef.Width TOPICS_WIDTH =            ConfigDef.Width.LONG;
   private static final String TOPICS_DOC =
       "List of topics to consume, separated by commas";
+  public static final String TOPICS_DEFAULT = "";
   private static final String TOPICS_DISPLAY =                   "Topics";
 
+  public static final String TOPICS_REGEX_CONFIG =                     "topics.regex";
```

Comment:
I don't see TOPICS_REGEX_CONFIG is used anywhere to retrieve topics. Is there a following commit?